### PR TITLE
fixed blocks with fixed height and width

### LINKED ISSUE #7: Make sure site looks the same with different texts and languages
There should be no width/height on some elements with text content

### PR DESCRIPTION
**Pull request by lyudmyla-horbyk**
Closes #7

### CODE REVIEW

**Comment by lyudmyla-horbyk** [comment]
@probil  Please, check the changes!

File: index.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="uk">
```
Comment (probil) [inline index.html:2]: it seems like those changes are not related to the fixed height and width

**Review by probil — CHANGES_REQUESTED** [review]
check comments

**Review by probil — APPROVED** [review]
